packages/react-native/ReactCommon/react/renderer/components/view/platform/macos/react/renderer/components/view/HostPlatformViewProps.h

```diff
@@ -32,12 +34,14 @@ class HostPlatformViewProps : public BaseViewProps {
       const char* propName,
       const RawValue& value);
 
-  MacOSViewEvents macOSViewEvents{};
+  HostPlatformViewEvents hostPlatformEvents{};
 
 #pragma mark - Props
 
   bool focusable{false};
   bool enableFocusRing{true};
 
+  std::vector<HandledKey> keyDownEvents{};
+  std::vector<HandledKey> keyUpEvents{};
```

Comment:
Should we add `#include <vector>` at the top?

Reply:
I'm going to assume this works in Xcode because we get the header transitively. Added the header.